3.15/localisation/de.json

```diff
@@ -0,0 +1,1771 @@
+{
+  "appName": "Optimize",
+  "appFullName": "Camunda Optimize",
+  "navigation": {
+    "analysis": "Analyse",
+    "events": "Eventbasierte Prozesse",
+    "logout": "Abmelden",
+    "logoutFailed": "Abmeldung fehlgeschlagen",
+    "logoutSuccess": "Sie haben sich erfolgreich abgemeldet.",
+    "configLoadingError": "Beim Laden der UI-Konfigurationen ist ein Fehler aufgetreten",
+    "documentation": "Dokumentation",
+    "help": "Hilfe",
+    "profile": "Profil",
+    "appSwitcher": "App Switcher",
+    "dashboards": "Dashboards",
+    "collections": "Sammlungen",
+    "apps": {
```

Comment:
This is no longer part of the translation files in https://github.com/camunda/camunda-optimize/blob/master/optimize/backend/src/main/resources/localization/en.json.